starters/nuxt-2/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "nuxt-2",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt start",
+    "generate": "nuxt generate",
+    "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
+    "lint": "npm run lint:js",
+    "lintfix": "npm run lint:js -- --fix"
+  },
+  "dependencies": {
+    "@nacelle/shopify-checkout": "^0.0.7",
+    "@nacelle/storefront-sdk": "^1.0.2",
+    "core-js": "^3.19.3",
+    "fuse.js": "^6.5.3",
+    "idb-keyval": "^6.1.0",
+    "nuxt": "^2.15.8",
+    "uuid": "^8.3.2",
+    "vue": "^2.6.14",
+    "vue-server-renderer": "^2.6.14",
+    "vue-template-compiler": "^2.6.14",
+    "webpack": "^4.46.0"
```

Comment:
```suggestion
```

`vue-server-renderer`, `vue-template-compiler`, and `webpack` look like extraneous dependencies. For reference, here's the `package.json` of a fresh Nuxt 2 project created with `npm init nuxt-app`, and some pretty minimal CLI options chosen:

```json
{
  "name": "new-nuxt-app",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "nuxt",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "nuxt generate",
    "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
    "lint": "npm run lint:js"
  },
  "dependencies": {
    "@nuxtjs/pwa": "^3.3.5",
    "core-js": "^3.15.1",
    "nuxt": "^2.15.7"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.14.7",
    "@nuxtjs/eslint-config": "^6.0.1",
    "@nuxtjs/eslint-module": "^3.0.2",
    "eslint": "^7.29.0",
    "eslint-config-prettier": "^8.3.0",
    "eslint-plugin-nuxt": "^2.0.0",
    "eslint-plugin-vue": "^7.12.1",
    "prettier": "^2.3.2"
  }
}
```

<img width="650" alt="package json npm init nuxt-app" src="https://user-images.githubusercontent.com/5732000/150703950-d1a858f3-0ae4-48ff-8207-7f62fbc09021.png">